src/format.js

```diff
@@ -122,22 +122,23 @@ const formatMosaics = mosaics => {
 }
 
 // FORMAT MOSAICS INFO
-const formatMosaicInfo = (mosaicInfo, mosaicName) => {
-  let mosaicObj = {
-    mosaic: mosaicName.mosaicId.toHex(),
-    namespace: mosaicName.names[0]?.name,
-    divisibility: mosaicInfo.divisibility,
-    address: mosaicInfo.owner.address.plain(),
-    supply: mosaicInfo.supply.compact(),
-    revision: mosaicInfo.revision,
-    startHeight: mosaicInfo.height.compact(),
-    duration: mosaicInfo.duration.compact(),
-    supplyMutable: mosaicInfo.flags.supplyMutable,
-    transferable: mosaicInfo.flags.transferable,
-    restrictable: mosaicInfo.flags.restrictable
-  }
-
-  return mosaicObj
+const formatMosaicInfo = mosaicInfo => ({
```

Comment:
i think should include with mosaicName as well.
so we can get MosaicIdHex and which namespace there are belong.

Reply:
@AnthonyLaw That's not in the mosaic info model, however. We should have a function named `formatMosaicNames` or something similar, since the mosaic names uses something very different than the `MosaicInfo` model. In the `Mosaics` view, this would require a separate API call currently to get he name for each mosaic, and it's also fairly convoluted code to read (would take me a few days to actually get it running).